elintegrostartapp/grails-app/controllers/com/elintegro/userProfile/ProfileDetailController.groovy

```diff
@@ -47,4 +47,9 @@ class ProfileDetailController {
             log.error("Image doesn't exist")
         }
     }
+    def termAndConditions(){
+        def params = request.getJSON()
+        println(params)
+        render(view:"/termsAndCondition/termsAndCondition")
```

Comment:
how about opening as modal or a section in the single page application itself.

Reply:
it is now solved using v-html inside our layout descriptor